meta-oe/recipes-connectivity/openssh/openssh_%.bbappend

```diff
@@ -0,0 +1,14 @@
+do_install:append () {
+  if ${@bb.utils.contains("DISTRO_FEATURES", "hiba", "false", "true", d)} ; then
+    echo "DISTRO_FEATURES does not support hiba. Skip the installations"
+    exit 0
+  fi
+
+  install -d ${D}/usr/include/ssh
+  install -d ${D}/usr/include/ssh/openbsd-compat
+  install -d ${D}/usr/lib/
```

Comment:
Use bitbake variables instead of hardcoding paths where ever possible.

Reply:
Done. And moving to poky recipe